.travis.yml

```diff
@@ -115,7 +115,7 @@ jobs:
           stage: Test docs, astropy dev, and without optional dependencies
           env: PYTHON_VERSION=3.8
                CONDA_DEPENDENCIES='requests beautifulsoup4 html5lib keyring'
-               PIP_DEPENDENCIES='https://github.com/keflavich/httpbin/archive/master.zip pytest-astropy'
+               PIP_DEPENDENCIES='https://github.com/keflavich/httpbin/archive/master.zip pytest-astropy pyvo'
```

Comment:
This travis job is for having only the bare minimum, mandatory dependencies. `pyvo` is still "optional" only, so please do not add it here.

A separate question whether we switch it to be a requirement. We said we would do that once the gaia module moved on to use it, too, however I don't know any ETA for that, so maybe we should just make that move without waiting any longer. I'll open a separate issue for the discussion.

Reply:
Hmm. I was under the impression that mandatory dependencies are listed in `setup.py`. To run the cadc tests I've had `pyvo` specified in `PIP_DEPENDENCIES` before. And it's still there for Python other than version 3.8. Hence my update.

Reply:
yes, but in this job we only install the mandatory ones to make sure we indeed don't always rely on optional ones. Naturally it means that for this job no cadc tests are run due to the pyvo dependency.

Reply:
They are both called `PIP_DEPENDENCIES` hence the confusion. I've removed it from P3.8 now.